dashboard/test/models/ability_test.rb

```diff
@@ -410,6 +431,8 @@ class AbilityTest < ActiveSupport::TestCase
     assert ability.cannot?(:read, @login_required_migrated_lesson)
     assert ability.cannot?(:student_lesson_plan, @login_required_migrated_lesson)
 
+    assert ability.can?(:read, @reference_guide)
```

Comment:
unsure if this one should be `cannot?` it seems like an admin would still be able to view reference guides (esp if logged out can as well)

Reply:
Good call out. Looks like you might need to also update here: https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/models/ability.rb#L408